src/std/build-spec.ss

```diff
@@ -312,13 +312,23 @@
     "net/httpd/api"
     "net/httpd"
     "net/sasl"
+    ;; :std/sxml
+    "sxml/print"
+    (gxc: "sxml/ssax")
+    "sxml/sxpath"
+    "sxml/xml"
+    "sxml/sxml"
+    (gxc: "sxml/html/parser" (extra-inputs: ("sxml/html/html-parser.scm")))
+    "sxml/tal/toplevel"
+    "sxml/tal/iter"
+    "sxml/tal/expander"
+    "sxml/tal/syntax"
+    "sxml"
+    ;; :std/tal
+    "tal"
+    ;:std/html
+    "html"
```

Comment:
we can probably not include those, they are fine under markup -- note that std/xml is only retained for backwards compatibility and we make even want to remove it for v0.19